src/backend/joanie/core/filters/admin.py

```diff
@@ -70,6 +74,55 @@ def filter_by_query(self, queryset, _name, value):
         """
         return queryset.filter(translations__title__icontains=value).distinct()
 
+    def filter_by_state(self, queryset, _name, value):
+        """
+        Filter resource by looking for states that match the one provided
+        in the "state" parameter
+        """
+        now = django_timezone.now()
```

Comment:
for coherent in this method, maybe you should always use "lte" and "gt"?
Here you sometimes use "lt", "lte", "gt" and "gte" with no apparent coherence?

Reply:
The reason why I use equality or not is to be the closest possible to the conditions in `compute_state`
But fidelity to the second might be unnecessary, i'll change it to use the same equality each time